firebase-appdistribution/src/main/res/values/strings.xml

```diff
@@ -13,40 +13,37 @@
+  <string name="update_version_available" translation_description="Version string (and code) of the available update">Version %s (%d) is available.</string>
+  <string name="update_release_notes" translation_description="Text prefixing release notes">Release notes:</string>
+  <string name="update_yes_button" translation_description="Text on button to start update">Update</string>
+  <string name="update_no_button" translation_description="Text on button to cancel update">Cancel</string>
+  <string name="downloading_app_update" translation_description="Title of the notification showing during download of update">Downloading in-app update…</string>
+  <string name="download_completed" translation_description="Title of notification when download of update has completed">Download completed</string>
+  <string name="download_failed" translation_description="Title of notification when download of update has failed">Download failed</string>
+  <string name="install_failed" translation_description="Title of notification when installation of update has failed">Install failed</string>
+  <string name="notifications_group_name" translation_description="Title of the notification channel group for the SDK">Firebase App Distribution</string>
+  <string name="app_update_notification_channel_name" translation_description="Name of the notification channel for in-app updates">App updates</string>
+  <string name="app_update_notification_channel_description" translation_description="Description of the notification channel for in-app updates">Shows progress of in-app updates</string>
+  <string name="feedback_launch_failed" translation_description="Toast explaining that launching the feedback functionality failed">Failed to launch feedback</string>
+  <string name="feedback_unidentified_release" translation_description="Toast explaining feedback can't be provided, because release can't be found">Release not found. This app may not have been installed by App Distribution, or you may no longer have access.</string>
+  <string name="feedback_no_release" translation_description="Toast explaining that feedback would have been sent, but was not, since development mode was active">Would have sent feedback, but did not due to development mode.</string>
+  <string name="feedback_notification_channel_name" translation_description="Name of the notification channel for in-app feedback">Feedback</string>
+  <string name="feedback_notification_channel_description" translation_description="Description of the notification channel for in-app feedback">Shows notification to leave feedback about app</string>
+  <string name="feedback_notification_title" translation_description="Title of notification encouraging testers to leave feedback">We want your feedback!</string>
+  <string name="feedback_notification_text" translation_description="Text of notification encouraging testers to leave feedback">Tap to leave feedback for %1$s</string>
```

Comment:
Nit: how about "... to leave feedback **for the app**" just so it's clear what the placeholder represents.